experimental/client-app/README.md

```diff
@@ -18,12 +18,61 @@ Within your terminal, open one tab to run the proxy and another to run the app.
 ```bash
 npm install
 npm build 
-npm run dev
+npm run start
 ```
 
-- `server` runs the proxy server to target api on port `3000`
-- `start` runs your app for development on port `8000`, reloading on file changes
-- `dev` uses npm-run-all to run the above `server` and `start` scripts in parallel (Perfect for cloudshell development.)
+To run locally:
+
+- `npm start` runs your app for development on port `8000`, reloading on file changes
+
+## Deploying Lit container
+
+```
+// Requires the following environment
+// variables to be set Cloud Run container
+REDIRECT_URI - uri pointing to auth api callback (i.e */auth/google)
+API_URL - url that points to server api to obtain cause/campaign information
+SITE_URL - url that points to current client app (this app)
+AUTH_API_URL - url that points to nodejs auth api url
+CLIENT_ID - oauth client id
+CLIENT_SECRET - oauth client secret
+```
+
+```bash
+export PROJECT_ID = your-project-id
+gcloud builds submit --tag gcr.io/$PROJECT_ID/lit-auth-website
+gcloud run deploy --image=gcr.io/$PROJECT_ID/lit-auth-website --port 8000
+gcloud run services update lit-auth-website \
+--update-env-vars REDIRECT_URI=$(your-redirect-uri),API_URL=$(your-api-url),AUTH_API_URL=$(your-auth-api-url),SITE_URL=$(your-client-site-url) \
+--update-secrets CLIENT_ID=$(your-client-id),CLIENT_SECRET=$(your-client-secret)
```

Comment:
Totally fine with me, but my suggestion for clarity is to use the "repeating flag" pattern instead of comma-delimiters:

```suggestion
gcloud run services update lit-auth-website \
  --update-env-vars REDIRECT_URI=$(your-redirect-uri) \
  --update-env-vars API_URL=$(your-api-url) \
  --update-env-vars AUTH_API_URL=$(your-auth-api-url) \
  --update-env-vars SITE_URL=$(your-client-site-url) \
  --update-secrets CLIENT_ID=$(your-client-id) \
  --update-secrets CLIENT_SECRET=$(your-client-secret)
```

I find this approach has some overhead in repeating the flags, but it's easier for the reader to process each property involved.